example-shoppe-ios/example-shoppe-iosTests/example_shoppe_iosTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  example_shoppe_iosTests.swift
+//  example-shoppe-iosTests
+//
+//  Created by Harold Davis Jr. on 2/24/20.
+//  Copyright © 2020 Harold Davis Jr. All rights reserved.
+//
+
+import XCTest
+@testable import example_shoppe_ios
+
+class example_shoppe_iosTests: XCTestCase {
```

Comment:
This is a good time to come to closure on the repo name. There was some conversation in the [CH card](https://app.clubhouse.io/fullstory/story/104413/ask-josh-humphries-to-setup-repos-on-fullstorydev-github) between myself, @van-fs and @jhump about whether the repo names should change.

@van-fs would you mind reviewing that thread again and adding any additional thoughts after my latest comment?

Reply:
ok sounds good